domains/certificates/Shared/DataContext/Migrations/20240423100351_ModifyWalletUrlsOnContracts.cs

```diff
@@ -0,0 +1,23 @@
+using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace DataContext.Migrations
+{
+    /// <inheritdoc />
+    public partial class ModifyWalletUrlsOnContracts : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.Sql(
+                "UPDATE public.\"Contracts\" SET \"WalletUrl\" = REPLACE(\"WalletUrl\", '/wallet-api', '/v1/slices');");
+        }
+
+        /// <inheritdoc />
+        protected override void Down(MigrationBuilder migrationBuilder)
+        {
+
```

Comment:
mangler der en invers operation i down?

Reply:
Nej - migrationen skal fjernes igen når den har kørt

Reply:
Ikke forstået, en migration har vel en up og en down metode af en grund?

Reply:
Det har du for så vidt ret i, men jeg kan ikke se, down metoden blive brugt noget sted